pkgs/applications/editors/emacs/elisp-packages/manual-packages/calfw/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  trivialBuild,
+  howm,
+}:
+
+trivialBuild {
```

Comment:
Please use `melpaBuild` like other packages in `manual-packages` dir.  See #278925.

BTW, I am wondering why `trivialBuild` seems to be used more.

Reply:
While I noticed that other packages in the dir used `melpaBuild`, I assumed (confessing that I did not spend time to look at its source) that it was for packages in MELPA that needed special attention. `trivialBuild` just sounded more like a default, so that might be the reason?